setup.py

```diff
@@ -36,10 +36,15 @@ def get_requirements():
 
 data_files = {}
 
+# https://packaging.python.org/guides/single-sourcing-package-version/
+version = {}
+with open("./conu/version.py") as fp:
```

Comment:
Please, add this also to docs/source/conf.py file (https://github.com/fedora-modularity/conu/blob/master/docs/source/conf.py#L59)

Reply:
I sure can!